test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -178,7 +203,11 @@ void test_ucp_peer_failure::fail_receiver() {
     // TODO use force-close to close connections
     flush_worker(failing_receiver());
     m_failing_memh.reset();
-    failing_receiver().cleanup();
+    {
+        /* hide warning messages about unreleased rdescs */
```

Comment:
don't hide, just transform to INFO message

Reply:
sorry, wrong comment
the are transformed to the `UCS_TEST_MESSAGE`